bitnami/elasticsearch/templates/metrics-deploy.yaml

```diff
@@ -43,10 +43,10 @@ spec:
           args:
             {{- if gt (int .Values.coordinating.replicas) 0 }}
             # Prefer coordinating only nodes to do the initial metrics query
-            - --es.uri=http://{{- if .Values.security.enabled }}elastic:{{ .Values.security.elasticPassword }}@{{- end }}{{ template "elasticsearch.coordinating.fullname" . }}:{{ .Values.coordinating.service.port }}
+            - --es.uri=http://{{- if .Values.security.enabled }}elastic:{{ .Values.security.elasticPassword }}@{{- end }}{{ template "elasticsearch.coordinating.fullname" . }}:9200
```

Comment:
Hi @jaudiger,

Could you please revert the change in this line?
The use of `:{{ .Values.coordinating.service.port }}` is fine, since it references the REST API port.

The problem here is that `{{ .Values.master.service.port }}` does not reference the Rest port but the Transport port, and the value 9200 is hardcoded in the master service, therefore it is okay to also hardcode here in line 49.